### ISSUE TITLE
Being in huge coordinates with the Supplementaries mod causes a enormous amount of lag spike

### ISSUE THREAD

**Issue by DarSitam** [henkelmax/sound-physics-remastered#263]
### Bug description

When this mod is installed alongside Supplementaries for Forge, going to coordinates around the millions can causes a lot of lag spikes.
The bigger the coordinates, the longer the lag spikes are.
Not having Supplementaries installed stops the issue. Setting the "player" sound slider to 0 also fixes the issue

### Minecraft version

1.20.1

### Mod version

1.20.1-1.4.5

### Mod loader and version

Forge 47.3.12

### Steps to reproduce

- Install Supplementaries (version 3.1.11), Moonlight lib (version 2.13.34)
- Generate any world (I'd recommend a superflat)
- Teleport to 29000000 ~ ~

### Expected behavior

You should now experience some heavy lag spikes. After this, you can set the player sound slider to 0 to stop these lags

### Log files

https://gist.github.com/DarSitam/136b03519c29dba7980c9756267ea9ce

### Videos

https://youtu.be/Xg9ChWXLvV0?si=3RSSLI6cvZZp3j8y
The video recorded the session described with the log files

**Comment by henkelmax** [comment]
Please update the mod to the latest version.

**Comment by DarSitam** [comment]
Seems like the 1.21.1 version fixed the issue. 
Since I use a modpack, I can't really upgrade to the latest version easily, but i'll see what I can do with the provided config.
I'll close the issue since it's fixed

**Comment by henkelmax** [comment]
> Please update the mod to the latest version.

I meant the mod version, not the Minecraft version.

**Comment by DarSitam** [comment]
Yeah, only those 2 mods. Disabling one of those stop the lags

**Comment by henkelmax** [comment]
And does it happen without SPR aswell?

**Comment by DarSitam** [comment]
Theres no issue without SPR OR Supplementaries. The issue is present only when the two mods are loaded.

BTW, here's two ways to fix the issue : 

- Setting the "players" sound slider to 0
- In "config\sound_physics_remastered\allowed_sounds.properties", find the line "supplementaries\:block.rope.slide" line and set it to false

**Comment by henkelmax** [comment]
That sounds like an issue with supplementaries then. They might spam that sound a lot or something. Please report that issue to their issue tracker.

**Comment by DarSitam** [comment]
Already did [here](https://legacy.curseforge.com/minecraft/mc-mods/supplementaries/issues/1059), and they told me to open an issue on your mod instead

**Comment by henkelmax** [comment]
I looked into the code and did a little bit of debugging. The issue is caused by supplementaries playing the sound near the 0, 0 coordinates which is millions of blocks away from the player. They are also constantly playing the sound at the player which doesn't make any sense.

I added a config option that ignores sounds that are played very far away, which solves this issue. But it was definitely an issue with their mod.
